Documentation/Diagnostics/PH2014.md

```diff
@@ -0,0 +1,34 @@
+# PH2014: Avoid Owner attribute
+
+| Property | Value  |
+|--|--|
+| Package | [Philips.CodeAnalysis.MsTestAnalyzers](https://www.nuget.org/packages/Philips.CodeAnalysis.MsTestAnalyzers) |
+| Diagnostic ID | PH2014 |
+| Category  | [Maintainability](../Maintainability.md) |
```

Comment:
MsTest Category ??

Reply:
File an Issue to consider converting all these to an MsTest Category. I'm thinking possibly not because it's already "categorized" by a separate package.

Reply:
Well, in `SuppressMessage` you only have the ID and the Category (of which the category is ingnored by now)

Reply:
Ahhh...